argocd/iac/terraform/examples/eks/private-git/outputs.tf

```diff
@@ -0,0 +1,33 @@
+output "configure_kubectl" {
+  description = "Configure kubectl: make sure you're logged in with the correct AWS profile and run the following command to update your kubeconfig"
+  value       = <<-EOT
+    export KUBECONFIG="/tmp/${module.eks.cluster_name}"
+    aws eks --region ${local.region} update-kubeconfig --name ${module.eks.cluster_name}
+  EOT
+}
+
+output "configure_argocd" {
+  description = "Terminal Setup"
+  value       = <<-EOT
```

Comment:
that's the normal print out
you can do the raw output to not get the EOT

Reply:
since I work with multiple cluster in different terminal windows in vscode I use this
```shell
export KUBECONFIG="/tmp/ex-private-git"
aws eks --region eu-west-1 update-kubeconfig --name ex-private-git
echo "ArgoCD URL: https://$(kubectl get svc -n argocd argo-cd-argocd-server -o jsonpath='{.status.loadBalancer.ingress[0].hostname}')"
echo "ArgoCD Username: admin"
echo "ArgoCD Password: $(kubectl get secrets argocd-initial-admin-secret -n argocd --template="{{index .data.password | base64decode}}")"
```

Reply:
the other ones if you just need `kubectl` and the other one if you want to configure `argocd` login and port forward